backend/backends/graphite/graphite.go

```diff
@@ -85,6 +85,10 @@ type client struct {
 	legacyNamespace  bool
 }
 
+func (client *client) Run(ctx context.Context) error {
+	return client.sender.Run(ctx)
+}
```

Comment:
As you know interfaces are used in the "consumer" bit of code, not when methods are defined (unlike e.g. Java). I could have done the following as some projects do but I think this is not idiomatic because this is by design that you do not specify which interfaces you "implement".
```go
var _ backendTypes.RunnableBackend = &client{}
```